skills/writing-documentation/SKILL.md

```
@@ -0,0 +1,351 @@
+```
+
+CORRECT: `@walkeros/collector` (with backticks) WRONG: @walkeros/collector (no
+backticks)
+
+```
+
+### Domain References
+
+```
+
+CORRECT: www.walkeros.io or relative paths DO NOT USE: legacy domain references
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)